src/v2/view-builder/utils/LinksUtil.js

```diff
@@ -4,7 +4,8 @@ import { ACTIONS, FORMS as RemediationForms } from '../../ion/RemediationConstan
 const { ENROLLED_PASSWORD_RECOVERY_LINK, ORG_PASSWORD_RECOVERY_LINK } = ACTIONS;
 
 const getSwitchAuthenticatorLink = (appState) => {
-  if (appState.hasRemediationObject(RemediationForms.SELECT_AUTHENTICATOR_AUTHENTICATE)) {
+  if (appState.hasRemediationObject(RemediationForms.SELECT_AUTHENTICATOR_AUTHENTICATE) &&
+    appState.hasMoreThanOneAuthenticatorOption(RemediationForms.SELECT_AUTHENTICATOR_AUTHENTICATE)) {
```

Comment:
Since the new method `hasMoreThanOneAuthenticatorOption` checks if this form name exists in the remediation object, we should be able to condense this here and below to the latter part of the conditional.